### ISSUE TITLE
About the python version and core dumped error

### ISSUE THREAD

**Issue by Kirito0816** [mzweilin/EvadeML-Zoo#12]
Hi, 

Sorry to disturb you. I tried to run the code on python 3.5. But the numpy version 1.13.3 requires python 3.7 and the tensorflow-gpu 1.1.0 requires python 3.5 3.6 or 2.7. So, I tried python3.5 +tensorflow-gpu 1.1.0 +numpy 1.10.0, which results in a core dumped error when running the code. May I ask the exact python version of implementation? Thank you.

**Comment by Kirito0816** [comment]
I tried to use python 2 as is mentioned as the development environment. But the the numpy version 1.13.3 requires python 2.6.x and the tensorflow-gpu 1.1.0 requires 2.7.

**Comment by yaozhiang119** [comment]
你好，我也在尝试运行这个代码，您运行起来了吗，如果运行起来了可以告诉我一下运行的各种库的详细版本吗

**Comment by Kirito0816** [comment]
> 你好，我也在尝试运行这个代码，您运行起来了吗，如果运行起来了可以告诉我一下运行的各种库的详细版本吗

No, I tried with the latest python and tensorflow version which requires some code modification. But it turns out that there are many files/libraries missing.
